controllers/api/user-routes.js

```diff
@@ -1,28 +1,30 @@
 const router = require('express').Router();
-const { User } = require('../../models');
+const { User, User_State, Role, Team } = require('../../models');
```

Comment:
You don't appear to be using these new Models

Reply:
Removed the excess models from user routes